pkg/storaged/storage-controls.jsx

```diff
@@ -249,3 +250,29 @@ export class StorageUsageBar extends React.Component {
         );
     }
 }
+
+export class StorageMenuItem extends React.Component {
+    render() {
+        return <li><a onClick={this.props.onClick}>{this.props.children}</a></li>;
+    }
+}
+
+export class StorageBarMenu extends React.Component {
+    render() {
+        function toggle(excuse) {
+            const classes = "btn btn-primary" + (excuse ? " disabled" : "");
+            return (
+                <button className={classes} type="button" data-toggle="dropdown">
```

Comment:
This has accessibility related issues. Please add text for screen readers.

Reply:
I have added `aria-label="Create Devices"`.  Is that ok?

Reply:
You can also translate it :) `aria-label={_("Create Devices")}` (although I am still not absolutely sure if we should translate those, but I guess why not?)

Reply:
I think @Gundersanne asked if we wanted to do that for the Firewall accessibility fixes. I think we landed on Yes, since you want to be able to use Cockpit if you're both visually impaired and don't speak English.